app/kuma-dp/cmd/run.go

```diff
@@ -185,8 +186,8 @@ func newRunCmd(opts kuma_cmd.RunCmdOpts, rootCtx *RootContext) *cobra.Command {
 				DNSPort:             cfg.DNS.EnvoyDNSPort,
 				EnvoyVersion:        *envoyVersion,
 				Workdir:             cfg.DataplaneRuntime.SocketDir,
-				AccessLogSocketPath: core_xds.AccessLogSocketName(cfg.DataplaneRuntime.SocketDir, cfg.Dataplane.Name, cfg.Dataplane.Mesh),
-				MetricsSocketPath:   core_xds.MetricsHijackerSocketName(cfg.DataplaneRuntime.SocketDir, cfg.Dataplane.Name, cfg.Dataplane.Mesh),
+				AccessLogSocketPath: accessLogSocketName(cfg.DataplaneRuntime.SocketDir, cfg.Dataplane.Name, cfg.Dataplane.Mesh),
```

Comment:
If we are changing this we should do more than just changing how we build the name here. We already sent `Workdir` with bootstrap request. We should build the name in the [policy](https://github.com/kumahq/kuma/blob/af83aed08e98d7526f711d6dc8197d4d0cbf62ca/pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin.go#L47) using workdir. Just like we do it in [MeshMetric plugin](https://github.com/kumahq/kuma/blob/5e7b4f86e2b454131e707c3cb35ba740b5715573/pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go#L159). This applies to access log and old metrics socket

Reply:
I totally got your point.
There's a concern that actually we just wanna keep the `accessLogSocketName` function in **private** within kuma-dp rather than keep it public in our pkg dir.. If we do the thing like 
```go
SocketName:   openTelemetrySocketName(proxy.Metadata.WorkDir, backendName),
```

We may have to keep `accessLogSocketName` to public and locate it in our `pkg/core/sockets.go` file again because both `kuma-dp` and `meshaccesslog policy` and other places do call the function. It might looks a little bit weird? 


My original intention is to follow the previous PR  [TODO](https://github.com/lahabana/kuma/blob/fd6ee1218f399f4e7d92e6681411d2810df8f83b/pkg/core/xds/sockets.go#L8-L18)